Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -69,6 +72,9 @@ ProjectData::ProjectData(ConfigTree const& project_config,
 
 	// output
 	parseOutput(project_config.get_child("output"), path);
+
+	// timestepping
+	parseTimeStepping(project_config.get_child("time_stepping"));
```

Comment:
Here, with a boolean return type, the program can determine whether the time stepping is needed. That means that the input of time stepping data can not be mandatory.  If there is not any time stepping data in the input file, the problem is solved as steady state one. Under such consideration, the change of the existing input files is not needed.